runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -144,10 +175,9 @@ public int hashCode() {
          * Obtain the {@link CorfuTable.IndexFunction} via its registered {@link CorfuTable.IndexName}.
          *
          * @param name name of the {@code IndexKey} previously registered.
-         * @param <I>  type of the {@code IndexKey}.
          * @return the instance of {@link CorfuTable.IndexFunction} registered to the lookup name.
          */
-        <I extends Comparable<?>> Optional<CorfuTable.IndexFunction<K, V, I>> get(CorfuTable.IndexName name);
+        Optional<CorfuTable.Index<K, V, ? extends Comparable<?>>> get(CorfuTable.IndexName name);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)